src/clj/game/cards/assets.clj

```diff
@@ -1777,13 +1778,13 @@
                           (continue-ability
                             (let [move-to target]
                               {:async true
-                               :prompt "Move how many tokens?"
+                               :prompt "Move how many advancement tokens?"
                                :choices {:number (req (get-counters card :advancement))
                                          :default (req (get-counters card :advancement))}
                                :cost [:trash]
                                :effect (effect (add-counter move-to :advancement target {:placed true})
                                                (system-msg (str "trashes Reconstruction Contract to move "
```

Comment:
Itself?

Reply:
We're still using `system-msg` here, so the log doesn't have the "Runner uses XXX to" prefix. Using "itself" would result in "The runner trashes *itself*", which would be awkward 😆